Moderation-Policy.md

```diff
@@ -165,10 +165,10 @@ of the [Code of Conduct][].
     is temporary or indefinite is required, along with a note justifying the
     action.
 * If an explanation of a Moderation action for a non-Collaborator Post is
-  provided, it must be provided in:
-  * The original Post being modified (as replacement or appended content),
-  * A new post within the original thread, or
-  * A new issue within the private nodejs/moderation repository.
+  provided, it must be provided in a new issue within the private
+  [nodejs/moderation][] repository. The reason for this is that we want to keep
+  a trace of the Moderation action even if the issue or pull request is deleted
```

Comment:
Pull requests cannot be deleted.

Is this relevant for any moderation action or is this really just for blocking?

Reply:
I'm not sure, I think only blocking makes sense.